### ISSUE TITLE
Enable swipe left / right to allow swiping through accounts on MainPage

### ISSUE THREAD

**Issue by setoy** [Top-Cat/UnofficialSteamAuthenticator#20]
Instead of having to go to "Switch account" and select an account, allow swiping left / right to swipe through accounts from the MainPage

**Comment by Top-Cat** [comment]
Why not both?

**Comment by setoy** [comment]
Yeah, I was talking about both. Keep the menu, but swiping is more convenient :)

**Comment by Top-Cat** [comment]
Dammit, I was editing...

...or configurable?
I've had problems in the past where it was too easy to swipe between views and it would personally get on my nerves.

Plus this might not mesh well with #21 or other future functionality, unless scoped to the steam guard section only. May be confusing to users for their chats to disappear from a swipe.

If limited to steam guard only would a swipe gesture (up?) to reach the selection screen be more useful as it shows all the steam guard tokens anyway.

**Comment by Top-Cat** [comment]
Or were you suggesting gestures in the list? That's certainly not happening.

**Comment by setoy** [comment]
Nope, not in the list, don't worry.
